ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml

```diff
@@ -67,6 +67,26 @@ tests:
     test:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws
+- always_run: false
+  as: udn-density-l3-pods-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      FEATURE_SET: TechPreviewNoUpgrade
+      IF_INSTALL_INFRA_WORKLOAD: "false"
```

Comment:
I would like for us to keep infra nodes w/ ingress/monitoring.

Reply:
I hit issues similar to this. [Slack Thread](https://redhat-internal.slack.com/archives/C0780C0J6HJ/p1727900717063329?thread_ts=1727900311.516419&cid=C0780C0J6HJ) 
Thanks for tagging me on it :)